A/Aeron/build_tarballs.jl

```diff
@@ -15,46 +15,61 @@ script = raw"""
 apk update
 apk add openjdk11 hdrhistogram-c-dev libbsd-dev util-linux-dev
 cd $WORKSPACE/srcdir/aeron
-mkdir build && cd build
-CMAKE_FLAGS=(-DCMAKE_INSTALL_PREFIX=$prefix
--DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
+mkdir -p /sys/devices/system/cpu/
+echo 1-${nproc} > /sys/devices/system/cpu/possible
```

Comment:
Uhm, why this?

Reply:
Which recipe?  Can you share the piece of code which is using that file?

Reply:
Looks like this is not needed in the end; thanks for catching it!